submissions/Zakkarat/html-css-popup/index.html

```diff
@@ -12,92 +12,92 @@
           <a href="#" class="refer">LMail</a>
           <a href="#" class="refer">Images</a>
           <input class="hideBoxes hide boxes" type="checkbox" id ="menu">
-          <label class="icons" for="menu"><img src="./images/icons.png"></label>
+          <label class="icons" for="menu"><img src="./images/icons.png" tabindex="0"></label>
```

Comment:
Remove `tabindex="0"` and use 
```
.boxes:focus + .icons {
    outline: auto;
}
```
Same story with button More.

Reply:
Thank you for advice! Done.